pages/Transfer.qml

```diff
@@ -311,7 +312,7 @@ Rectangle {
                   }
 
                   validator: RegExpValidator {
-                      regExp: /^(\d{1,8})?([\.]\d{1,12})?$/
+                      regExp: /^(\d{1,8})?([\.|,]\d{1,12})?$/
```

Comment:
Treats `1|1` as a valid amount

Reply:
Thanks, my regex sucks, please check my fix.

Reply:
Should be `^(\d{1,8})?([\.,]\d{1,12})?$`

Reply:
Ehh, yes. Thanks.